modules/microshift-blocking-nodeport-access.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-blocking-nodeport-access_{context}"]
+= Blocking external access to the NodePort service on a specific host interface
+
+OVN-Kubernetes does not restrict the host interface where a NodePort service can be accessed from outside a {product-title} node. The following procedure explains how to block the NodePort service on a specific host interface and restrict external access.
```

Comment:
Right, I was thinking that as well, lemme check with this bc I'm seeing it being used differently around the docs when talking about just NodePort vs the NodePort Service

Reply:
Got it. If this is an exception, maybe we can get some clarity, bring it up at the team meeting, and add it to the glossary.

Reply:
@kelbrown20 and @michaelryanpeter I just had this issue in a previous PR and my SME (@zshi-redhat) corrected "node ports" to NodePort service, so that's what we have been doing in the Networking portions of the MicroShift docs. https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.12/html/networking/microshift-applying-networking-settings

Reply:
That makes sense. According to the repo rules, if we are talking about the specific object, it should be `NodePort` service. But, if this is important to engineering and how customers expect to see it, it should be added to the repo guidelines glossary. Hopefully, that can reduce churn :)

Reply:
Just for reference, I suggested to use `NodePort` to align with the kubernetes service type defined here: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types, other service types include ClusterIP, LoadBalancer and ExternalName.

Reply:
Gotcha! Will update!